### ISSUE TITLE
[Genesis Plus GX] Ludo crashes while trying to load Sega SC-3000 computer games

### ISSUE THREAD

**Issue by fmaida** [libretro/ludo#438]
If you try to scan some Sega SC-3000 games/apps in `.sc` format, Ludo will wrongly recognize them as Sega SC-1000 games. Whenever you will try to launch one of those games, Ludo will try to open it with the default emulator set for SG-1000 games (Genesis Plus GX). But since Genesis Plus GX cannot handle Sega SC-3000 files, Ludo will crash (BlueMSX can handle these files, and maybe Kega Fusion too).

I'm attaching a log file: 
[sc-3000-Chuugaku-Hisshuu-Eibunpou.log.txt](https://github.com/libretro/ludo/files/6682649/sc-3000-Chuugaku-Hisshuu-Eibunpou.log.txt)

This is an example of SC-3000 file that will crash Ludo:
[Chuugaku Hisshuu Eibunpou (Chuugaku 1-Nen) (Japan) (SC-3000).zip](https://github.com/libretro/ludo/files/6682652/Chuugaku.Hisshuu.Eibunpou.Chuugaku.1-Nen.Japan.SC-3000.zip)

**Comment by kivutar** [comment]
OK, were you using scanning? Maybe our database is wrong.

**Comment by fmaida** [comment]
Yes, I used the scanner function. Both the SG-1000 and the SC-3000 roms share the same file extension (.sc). The problem is that Genesis Plus GX crashes while trying to load a SC-3000 cartridge under Ludo.

RetroArch in this case will correctly scan it and bind the rom to BlueMSX; if I try to manually load the SC-3000 rom, RetroArch will also prevent me from choosing Genesis Plus GX since the emulator cannot handle SC-3000 files.

**Comment by kivutar** [comment]
For now Ludo doesn't do any extension based scanning. It relies only on the database, checksums, and long filenames.

**Comment by fmaida** [comment]
Then probably someone have probably mixed SG-1000 and SC-3000 entries in the same database. I suppose that the easiest way to resolve this issue would be to switch the default core to BlueMSX that can load both, even though you would not have access to the keyboard in this case.

**Comment by kivutar** [comment]
Can you check here? They are xml files https://github.com/kivutar/ludo-database/tree/kivutar/xml

**Comment by kivutar** [comment]
OK, SC-3000 games are indeed listed in the SG-1000 db https://github.com/kivutar/ludo-database/blob/aad97888edaccdb9a37491ccd84453436b9e2e46/Sega%20-%20SG-1000.dat#L277

**Comment by fmaida** [comment]
Would be okay if I make you a PR that moves the SC-3000 entries to a new xml file?
